IndicoV2.IntegrationTests/Utils/DataHelpers/Workflows/WorkflowHelper.cs

```diff
@@ -19,10 +25,8 @@ public async Task<IWorkflow> GetAnyWorkflow()
             return workflows.First();
         }
 
-        public WorkflowHelper(IDataSetClient dataSets, IWorkflowsClient workflows)
-        {
-            _dataSets = dataSets;
-            _workflows = workflows;
-        }
+        public async Task<IWorkflow> Get(bool autoReviewEnabled) => 
+            // TODO: implement autoReviewEnabled
```

Comment:
Should this be implemented?

Reply:
This should be done after we have CI working (this method should be able to return workflow with autoReviewEnabled=true).